src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -299,7 +308,8 @@
     short-summary: The shared access signature for the source storage account.
 examples:
   - name: Copy multiple blobs or files to a blob container. Use `az storage blob show` to check the status of the blobs. (autogenerated)
```

Comment:
I think here is not very accurate. all the things in a container should be blobs not files

Reply:
The is the old one. Not the purpose of this PR.